### ISSUE TITLE
Team C Feedback

### ISSUE THREAD

**Issue by tranvu21** [AaronCerty/Project2#2]
Hi Team F,
As a newbie in blockchain, the proposal is slightly abstract to us due to the occurrence of several specialized definitions (EVM, cross-chain, etc.). However, we merely get the idea of a dashboard that integrates data from different chains from your projects.
We have some suggestions for the proposal:
 - Besides the aforementioned metrics, it would be better if you specify what questions those illustrations can answer while other current solutions can not. I think this addition would emphasize your contribution to the field.
 - Are there any references for your idea ( maybe simple dashboards )? If yes, we suggest you consider adding them to the proposal, so that we can imagine what type of visualization, charts, etc. that you are going to use. Therefore, reviewers can judge the suitabilities of those proposed models for the given data or suggest better models.
 - As we observe, you divide the overall task into management, database handling, and frontend development. However, cause the project is data visualization, do you think that separation is fair enough? or should we prioritize on front end workload?
 
In conclusion, we appreciate what you are trying to contribute in this project. You might consider our feedback for further project development. Keep it up!

**Comment by AaronCerty** [comment]
Thank you for your valuable feedback on our proposal. Here’s how we plan to address your suggestions:

Clarifying Metrics and Their Impact: We will detail how our dashboard's metrics and visualizations provide unique insights and solve problems that current solutions do not.

Incorporating References and Examples: We will add examples of simple dashboards to our proposal to better illustrate the expected visualizations and charts.

Task Distribution and Frontend Development Focus: Acknowledging the project's emphasis on data visualization, we will adjust our task division to prioritize frontend development, ensuring effective data presentation.

We appreciate your insights and are excited to refine our proposal accordingly.

**Comment by tienvu95** [comment]
As also mentioned in my previous comment, our (initial) audience are COMP4010 classmates --> not many of us are familiar with the concepts introduced here. We should add additional information to make our dashboar user-friendly. For jargons in blockchain (like metrics), I think you can consider adding the explanation like the example attached. When you hover over a metrics, they can see its definition and formula.

![ảnh](https://github.com/AaronCerty/Project2/assets/26209350/f68fb529-2e23-452c-9dbc-e25f3d0f133a)

Regarding the referrence, we won't require it in your report if you can comprehensively explain your dashboard. The work allocation is up to you to decide, just make sure everyone is happy with it.

** Note that you don't have to reply to this comment, this is just my thought on what you can do to improve your project.**
